packages/mgt-components/src/components/mgt-people/mgt-people.scss

```diff
@@ -33,6 +33,13 @@ mgt-people .people-person {
 :host .overflow,
 mgt-people .overflow {
   span {
-    vertical-align: middle;
+    position: relative;
+    color: #1a1a1a;
```

Comment:
Is there a design token/variable for this color? 
It feels like explicitly setting the color here might make it harder to theme this and could cause a11y issues on a dark mode